src/example/Simplest.js

```diff
@@ -0,0 +1,71 @@
+import ActionManager from "../action/ActionManager.js";
+import DoubleJumpFilter from "./filters/DoubleJumpFilter.js";
+
+function main() {
+  // App specific setup of the ActionManager
+
+  let actionManager = new ActionManager();
+
+  // Load a custom filter
+  actionManager.loadFilter(DoubleJumpFilter, "/filter/double-jump");
+
+  // Load up the maps that bind inputs to filters and actions
+  actionManager
+    .loadActionMaps(
+      // Maps used while the mode is flat
+      "./flat-playing.map",
+      "./flat-paused.map",
+      "./flat-menu.map",
+      // Maps used while the mode is portal
```

Comment:
We may not want to separate action sets into different map files because whatever tool we come up with that allows rebinding controls and saving it to a `.map` file should probably let you rebind controls for all the action sets that are active during the lifetime of an application (in this case, `playing, paused, and menu`).

Reply:
I'd be careful about calling the extension `.map` - that extension is reserved for JavaScript source maps.

how about `.json` or `-mapping.json`?